.github/workflows/cd.yml

```diff
@@ -32,21 +32,20 @@ jobs:
       cli_version: ${{ steps.cli_version.outputs.result }}
 
     steps:
-
-    - id: api_version
-      name: dynamic input api version
-      run: |
-        if [[ "${{ inputs.api_version }}" == "" ]]; then echo dev; else echo ${{ inputs.api_version }}; fi
-
-    - id: worker_version
-      name: dynamic input worker version
-      run: |
-        if [[ "${{ inputs.worker_version}}" == "" ]]; then echo dev; else echo ${{ inputs.worker_version }}; fi
-
-    - id: cli_version
-      name: dynamic input cli version
-      run: |
-        if [[ "${{ inputs.cli_version }}" == "" ]]; then echo dev; else echo ${{ inputs.cli_version }}; fi
+      - id: api_version
+        name: dynamic input api version
+        run: |
+          if [[ "${{ inputs.api_version }}" == "" ]]; then echo dev >> $GITHUB_OUTPUT; else echo ${{ inputs.api_version }} >> $GITHUB_OUTPUT; fi
```

Comment:
Why do we save the `inputs.api_version` in the `GITHUB_OUTPUT` env variable?

Reply:
This is not the latest commit.
Best hide this comment.